### ISSUE TITLE
Need more details about groups in their descriptions

### ISSUE THREAD

**Issue by LMurphy001** [wmurphyrd/guppe#38]
Related to issue #22 https://github.com/wmurphyrd/guppe/issues/22 - Would it be possible to see the language(s) of the toots in the group on the group's description blurb? That way, if the group's toots are primarily in a language I won't understand, I don't need to join the group.

Also, maybe the initial toot which creates the group should have a short description of the purpose of the group.

**Comment by FediVideos** [comment]
Just had some feedback about this on FediTips, is there any way to add a description to each group so people know what it's about?

The automatically generated descriptions aren't enough in many cases.

**Comment by FediVideos** [comment]
Agreed, would it be possible to add descriptions somehow, so people know exactly what the topic is?

The automatically generated text isn't enough for all the groups.

**Comment by guerda** [comment]
Fully agree. Sometimes group's name is just too generic or too specific. I think a very short description would suffice.

Maybe it's possible to have it as simple text files in this repo which build the website.
